inveniordm_py/files/metadata.py

```diff
@@ -13,24 +13,69 @@
 class FileMetadata(Metadata):
     """File metadata."""
 
-    accept = "application/vnd.inveniordm.v1+json"
-    content_type = "application/vnd.inveniordm.v1+json"
+    accept = "application/json"
```

Comment:
Comment: I understand this is then content type needed right now for making requests to the files endpoint, so this is just a comment that In general all InvenioRDM REST APIs needs to be changed to use the ``application/vnd.inveniordm.v1+json`` content type so that we have proper versioning enabled, otherwise we cannot make any changes in the future without major hassle.

Reply:
I created a [ticket](https://github.com/inveniosoftware/inveniordm-py/issues/3) for this, to be addressed later when the RDM APIs are changed.